content/en/docs/reference/kubectl/overview.md

```diff
@@ -69,30 +69,30 @@ The following table includes short descriptions and the general syntax for all o
-`get`        | <code>kubectl get (-f FILENAME &#124; TYPE [NAME &#124; /NAME &#124; -l label]) [--watch] [--sort-by=FIELD] [[-o &#124; --output]=OUTPUT_FORMAT] [flags]</code> | List one or more resources.
-`label`        | <code>kubectl label (-f FILENAME &#124; TYPE NAME &#124; TYPE/NAME) KEY_1=VAL_1 ... KEY_N=VAL_N [--overwrite] [--all] [--resource-version=version] [flags]</code> | Add or update the labels of one or more resources.
+`expose`        | <code>kubectl expose (-f FILENAME &#124; TYPE NAME &#124; TYPE/NAME) `[--port=port]``[--protocol=TCP`&#124;`UDP]``[--target-port=number-or-name]``[--name=name]``[--external-ip=external-ip-of-service]``[--type=type]``[flags]`</code> | Expose a replication controller, service, or pod as a new Kubernetes service.
+`get`        | <code>kubectl get (-f FILENAME &#124; TYPE [NAME &#124; /NAME &#124; -l label]) `[--watch] [--sort-by=FIELD]` [[-o &#124; `--output]=OUTPUT_FORMAT] [flags]`</code> | List one or more resources.
+`label`        | <code>kubectl label (-f FILENAME &#124; TYPE NAME &#124; TYPE/NAME) KEY_1=VAL_1 ... KEY_N=VAL_N `[--overwrite]``[--all]``[--resource-version=version]``[flags]`</code> | Add or update the labels of one or more resources.
 `logs`        | `kubectl logs POD [-c CONTAINER] [--follow] [flags]` | Print the logs for a container in a pod.
-`patch`        | <code>kubectl patch (-f FILENAME &#124; TYPE NAME &#124; TYPE/NAME) --patch PATCH [flags]</code> | Update one or more fields of a resource by using the strategic merge patch process.
+`patch`        | <code>kubectl patch (-f FILENAME &#124; TYPE NAME &#124; TYPE/NAME) `--patch PATCH [flags]`</code> | Update one or more fields of a resource by using the strategic merge patch process.
 `port-forward`    | `kubectl port-forward POD [LOCAL_PORT:]REMOTE_PORT [...[LOCAL_PORT_N:]REMOTE_PORT_N] [flags]` | Forward one or more local ports to a pod.
 `proxy`        | `kubectl proxy [--port=PORT] [--www=static-dir] [--www-prefix=prefix] [--api-prefix=prefix] [flags]` | Run a proxy to the Kubernetes API server.
 `replace`        | `kubectl replace -f FILENAME` | Replace a resource from a file or stdin.
-`run`        | `kubectl run NAME --image=image [--env="key=value"] [--port=port] [--replicas=replicas] [--dry-run=server|client|none] [--overrides=inline-json] [flags]` | Run a specified image on the cluster.
-`scale`        | <code>kubectl scale (-f FILENAME &#124; TYPE NAME &#124; TYPE/NAME) --replicas=COUNT [--resource-version=version] [--current-replicas=count] [flags]</code> | Update the size of the specified replication controller.
+`run`        | <code>kubectl run NAME `--image=image``[--env="key=value"]``[--port=port]``[--dry-run=server`&#124;`client`&#124;`none]``[--overrides=inline-json]``[flags]`</code> | Run a specified image on the cluster.
```

Comment:
Hello @viniciusbds .
Have you rebased your branch recently?
There was a recent change (https://github.com/kubernetes/website/pull/20629) that corrected the rendering of double dashes.
I tested this file locally without the extra back slashes and the double dashes rendered correctly.
Would you recheck your changes? Thanks.

Reply:
Done 

Thanks for review @kbhawkey